examples/example/package-lock.json

```diff
@@ -1,115 +0,0 @@
-{
```

Comment:
I'll regenerate the package-lock files once the new version of es is up and can run npm i again